pylintrc

```diff
@@ -47,7 +47,7 @@ load-plugins=
 # --enable=similarities". If you want to run only the classes checker, but have
 # no Warning level messages displayed, use"--disable=all --enable=classes
 # --disable=W"
-#disable=
+disable=C0111
```

Comment:
oh yeah, also got tired of the "missing docstring" message in my linter. If anyone is unhappy with this decision I"m happy to remove this (and keep it in my global pylintrc).